src/av2/structures/ndgrid.py

```diff
@@ -142,16 +145,23 @@ def points_to_bev_img(
 
         points_xy = points[..., :2].copy()  # Prevent modifying input.
         indices = self.transform_to_grid_coordinates(points_xy)
```

Comment:
```suggestion
        indices_grid = self.transform_to_grid_coordinates(points_xy)
```

I wonder if we should be even more explicit here?